### PR TITLE
WIP: Handle resize event

### LINKED ISSUE #44: Resize handle feature
I am really missing the ability to resize the plot with the mouse like in the picture bellow with the grey bottom right corner arrow.

### PR DESCRIPTION
**Pull request by martinRenou**
This is supposed to fix #44 and #74. It should then be possible to resize the `figure.canvas` using ipywidgets layouts

### CODE REVIEW

File: js/src/mpl_widget.js [hunk 1]
```diff
@@ -25,6 +25,16 @@ var MPLCanvasView = widgets.DOMWidgetView.extend({
         this.fig = new mpl.figure(id, toolbar_items, this);
 
         this.el.appendChild(this.fig.root);
+    },
+
+    processPhosphorMessage: function(msg) {
+        MPLCanvasView.__super__.processPhosphorMessage.apply(this, arguments);
+        switch (msg.type) {
+        case 'resize':
+            var rect = this.el.getBoundingClientRect();
```
Comment (martinRenou) [inline js/src/mpl_widget.js]: This is not right, because `this.el` also contains the toolbar

**Comment by martinRenou** [comment]
Closing, I will open another PR for resizing support once #100 is in

**Comment by 01baftb** [comment]
@martinRenou Any estimate when this feature would be released?

**Comment by martinRenou** [comment]
Are you talking about a resize handle?

**Comment by 01baftb** [comment]
@martinRenou Yes I am referring to the resize handle feature as previously mentioned in this issue. https://github.com/matplotlib/jupyter-matplotlib/issues/44. It will also be cool if there was a input text box to specify the exact size instead of the mouse handle. 

Thank you for your help and contributions!

**Comment by martinRenou** [comment]
I have no estimate. I plan to look at it this week, but I cannot promise anything :)

**Pull request closed without merging**